admin/google_search_console/views/gsc-display.php

```diff
@@ -117,6 +117,17 @@
 		break;
 }
 ?>
+<?php
+        // Add link to Knowledge Base article about crawl issues.
+        echo '<p>';
+
+        printf(
+        /* translators: %1$s expands anchor to knowledge base article, %2$s expands to </a> */
+        __( 'Please refer to %1$sour article about how to connect your website to Google Search Console%2$s if you need assistance.', 'wordpress-seo'), '<a href="https://kb.yoast.com/kb/how-to-connect-and-retrieve-crawl-issues/" target="_blank">', '</a>' );
```

Comment:
Can you put a space before the closing function. `, 'wordpress-seo')` thus: `, 'wordpress-seo' )`